cranelift/interpreter/src/step.rs

```diff
@@ -1067,7 +1067,6 @@ where
             }
             assign(Value::int(result, ctrl_ty)?)
         }
-        Opcode::Vselect => assign(vselect(&arg(0)?, &arg(1)?, &arg(2)?, ctrl_ty)?),
```

Comment:
I see the remaining uses of this `vselect` helper are always fed by the result of `icmp`, to implement the min and max instructions. It'd be nice to clean that up. Opening a "good first issue" as a followup for this PR would be fine though.